XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -42,6 +38,78 @@ private void InitializeData()
+            MixedTypeRepeater.ItemsSource = basicData;
+
+            List<NestedCategory> nestedCategories = new List<NestedCategory>();
+            nestedCategories.Add(
+                new NestedCategory("Fruits",  new ObservableCollection<string>{
+                                                            "Apricots",
+                                                            "Bananas",
+                                                            "Grapes",
+                                                            "Strawberries",
+                                                            "Watermelon",
+                                                            "Plums",
+                                                            "Blueberries"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Vegetables", new ObservableCollection<string>{
+                                                            "Broccoli",
+                                                            "Spinach",
+                                                            "Sweet potato",
+                                                            "Cauliflower",
+                                                            "Onion",
+                                                            "Brussel sprouts",
+                                                            "Carrots"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Grains", new ObservableCollection<string>{
+                                                            "Rice",
+                                                            "Quinoa",
+                                                            "Pasta",
+                                                            "Bread",
+                                                            "Farro",
+                                                            "Oats",
+                                                            "Barley"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Proteins", new ObservableCollection<string>{
+                                                            "Steak",
+                                                            "Chicken",
+                                                            "Tofu",
+                                                            "Salmon",
+                                                            "Pork",
+                                                            "Chickpeas",
+                                                            "Eggs"
+                }));
+
+            outerRepeater.ItemsSource = nestedCategories;
+
+            // Set sample code to display on page's initial load
+            SampleCodeLayout.Value = "<muxc:StackLayout x:Name='VerticalStackLayout' Orientation='Vertical' Spacing='8'/>";
```

Comment:
These need to be double quotes instead of single. The purpose of the samples is allow people using the Gallery to copy-paste them into their own apps and use as is. So the sample code needs to show correct syntax.
```suggestion
            SampleCodeLayout.Value = "<muxc:StackLayout x:Name=\"VerticalStackLayout\" Orientation=\"Vertical\" Spacing=\"8\"/>";
```

Reply:
Interestingly, this only looks wrong when the page loads. If I click on one of the other layout radio buttons and go back to this layout, everything looks good. Are you loading the sample code from different places?

Reply:
Fixed. This code block populates the samples when the page loads, I must have missed it when replacing all the single quotes to double quotes.